src/askscript/__tests__/00-documentation-examples/documentation24-function_type.ask

```diff
@@ -1,5 +1,5 @@
 ask {
-  const f:string(int) = fun(n:int):string {
```

Comment:
Tests in 00-documentation-examples mirror exactly examples at the AskScript documentation.
If you remove the type here, please remove it also at https://www.notion.so/AskScript-Human-Friendly-Language-for-AskQL-7c5e382df87a4015b44985391861f6a2